MH4U Dex/DamageZoneCell.m

```diff
@@ -0,0 +1,57 @@
+//
+//  DamageZoneCell.m
+//  MH4U Dex
+//
+//  Created by Joseph Goldberg on 2/9/15.
+//  Copyright (c) 2015 Joseph Goldberg. All rights reserved.
+//
+
+#import "DamageZoneCell.h"
+
+@interface DamageZoneCell ()
+
+@property (weak, nonatomic) IBOutlet UILabel *bodyPartLabel;
+@property (weak, nonatomic) IBOutlet UILabel *cutLabel;
+@property (weak, nonatomic) IBOutlet UILabel *impactLabel;
+@property (weak, nonatomic) IBOutlet UILabel *shotLabel;
+@property (weak, nonatomic) IBOutlet UILabel *fireLabel;
+@property (weak, nonatomic) IBOutlet UILabel *waterLabel;
+@property (weak, nonatomic) IBOutlet UILabel *thunderLabel;
+@property (weak, nonatomic) IBOutlet UILabel *iceLabel;
+@property (weak, nonatomic) IBOutlet UILabel *dragonLabel;
+@property (weak, nonatomic) IBOutlet UILabel *koLabel;
+@property (weak, nonatomic) IBOutlet UILabel *extractLabel;
+
+@end
+
+@implementation DamageZoneCell
+
+- (void)displayContents
+{
+    self.bodyPartLabel.text = self.bodyPart;
+    self.extractLabel.text = self.extract;
+    if ([self.bodyPart isEqualToString:@"Body Part"]) {
```

Comment:
I'd like to later gut this out and replace it with a static header, so I'm leaving this as-is for now.